lib/tmuxinator/project.rb

```diff
@@ -147,6 +147,12 @@ def tmux_command
       yaml["tmux_command"] || "tmux"
     end
 
+    def tmux_has_session?(name)
+      sessions = `#{tmux_command} ls`
+
+      !sessions.match("^#{name}:")
```

Comment:
Err..., yeah that's just poor logic.  My bad - I was rushing to get this pushed so I could get back to work.

Fixed in next commit.